packages/neutron-sdk/src/interchain_queries/helpers.rs

```diff
@@ -164,7 +164,10 @@ pub fn get_update_time(commission: &Option<ValidatorCommission>) -> Option<u64>
 
 /// Returns denom for total supply from StorageValue key
 pub fn get_total_supply_denom(denom: &Binary) -> Option<String> {
-    if denom.len() > 1 {
+    if denom.len() <= 2 {
+        return None
+    }
+    if *denom.get(0)? == SUPPLY_PREFIX {
```

Comment:
```suggestion
    if denom.len() < 2 {
        return None
    }
    if denom[0] == SUPPLY_PREFIX {
```

Via switching from <= to < operator, we allow denoms of a single byte length.

Since we have ensured that length is always 2 or more, we don't have to access denom[0] securely, we can just access it as is.